keras_cv/models/resnet_v2_backbone.py

```diff
@@ -440,40 +350,25 @@ def __init__(
         )(x)
         x = layers.Activation("relu", name="post_relu")(x)
 
-        if include_top:
+        if pooling == "avg":
```

Comment:
here's an open question: should pooling belong to backbones or task models.

Reply:
When looking at the existing code it seems like pooling is only an option when `include_top=False`. Therefore I was a little confused when this option is actually used---maybe for another task like objection detection?

Either way, these layers have no weights so their presence does not impact preset loading.

Reply:
Talked offline. Removed the `pooling` arg to reduce conflicts with `Task` model behavior.